lib/relationship_ancestry_converter_helper.rb

```diff
@@ -0,0 +1,105 @@
+module RelationshipAncestryConverterHelper
```

Comment:
Are you planning on using this for other tables in future migrations?  If not, might just be better to embed this module directly into the migration that uses it.

Reply:
Yes. We're planning on using this for https://github.com/ManageIQ/manageiq-schema/pull/497 as well.

Reply:
Is it possible to write specs for it as a standalone module or is that kind of impossible?

Reply:
it is going to be used 3 times max? I'd honestly prefer copy/past over spending the money/time on testing this as an individual component. The output of the code is already tested for this particular use case. and will be tested for the other use case as well.